go.mod

```diff
@@ -4,31 +4,31 @@ go 1.13
 
 require (
 	github.com/go-logr/logr v0.1.0
-	github.com/go-openapi/spec v0.17.2
```

Comment:
what about go.sum and vendor/?
is there any change to update in the auto-generated code?